components/forms/Label/Label.tsx

```diff
@@ -24,7 +24,7 @@ export const Label = (props: LabelProps): React.ReactElement => {
 
   return (
     <label data-testid="label" className={classes} htmlFor={htmlFor}>
-      {children}
+      {children} {classes.includes("required") ? <span aria-label="required">*</span> : null}
```

Comment:
Happy to merge it the `className` detection, just wondering if it's worth passing the `required` prop to the Label component, as the corresponding fields get it too, e.g. https://github.com/cds-snc/platform-forms-client/blob/main/components/forms/TextInput/TextInput.tsx#L14

Reply:
Might be better to have it match the other components. I can move it over!

Reply:
Thank you!